<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="General">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="lblGeneral">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Connection">
        <attribute name="title">
         <string>Connection</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lblConsumerKey">
           <property name="text">
            <string>Consumer Key</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="consumerKey"/>
         </item>
         <item>
          <widget class="QLabel" name="lblConsumerSecret">
           <property name="text">
            <string>Consumer Secret</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="consumerSecret"/>
         </item>
         <item>
          <widget class="QLabel" name="lblTokenValue">
           <property name="text">
            <string>Token Value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tokenValue"/>
         </item>
         <item>
          <widget class="QLabel" name="lblTokenSecret">
           <property name="text">
            <string>Token Secret</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tokenSecret"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnCheckConnection">
           <property name="text">
            <string>Check Connection</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
